supervisor/homeassistant/api.py

```diff
@@ -124,12 +133,7 @@ async def check_api_state(self) -> bool:
 
         # Check if API is up
         with suppress(HomeAssistantAPIError):
-            async with self.make_request("get", "api/config") as resp:
-                if resp.status in (200, 201):
-                    data = await resp.json()
-                    if data.get("state", "RUNNING") == "RUNNING":
-                        return True
-                else:
-                    _LOGGER.debug("Home Assistant API return: %d", resp.status)
-
+            data = await self.get_config()
+            if data and data.get("state") == "RUNNING":
```

Comment:
```suggestion
            if data and data.get("state", "RUNNING") == "RUNNING":
```
That is for older HA installation